### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by surajkota**
Issue templates for bug reports, feature requests and documentation updates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (mbaijal) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Nit: Not a fan of the negative connotation with the example `Ex. I'm always frustrated when` :D

Comment (mbaijal) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: For a feature/ and maybe also a bug we could ask if this is a blocker.

**Review by mbaijal — APPROVED** [review]
Thanks for this PR !

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (surajkota) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: Will change, some of these blobs are Github defaults

**Review by rrrkharse — APPROVED** [review]

**Review by ryansteakley — APPROVED** [review]

**Pull request merged**